src/test/resources/application.properties

```diff
@@ -2,26 +2,24 @@
 server.port=8080
 
 ## Spring DATASOURCE (DataSourceAutoConfiguration & DataSourceProperties)
-spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.url=jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1
 spring.datasource.driver-class-name=org.h2.Driver
 spring.datasource.username=sa
 spring.datasource.password=
 
 ## Spring Liquibase
 spring.liquibase.change-log=classpath:liquibase/db.changelog-master.xml
-spring.liquibase.url=jdbc:h2:mem:testdb
-spring.liquibase.user=sa
-spring.liquibase.password=sa
 
 ### Hibernate Properties
 # The SQL dialect makes Hibernate generate better SQL for the chosen database
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.H2Dialect
-spring.jpa.hibernate.ddl-auto=update
+# Enabling H2 Console: http://localhost:8080/h2-console
+spring.h2.console.enabled=true
```

Comment:
You might remove it. It was added only for debugging purpuses.

Reply:
Removed